lib/pbench/server/database/models/users.py

```diff
@@ -17,6 +16,7 @@ class User(Database.Base):
     last_name = Column(String(255), unique=False, nullable=False)
     password = Column(LargeBinary(500), nullable=False)
     registered_on = Column(DateTime, nullable=False)
+    bcrypt_log_rounds = Column(Integer, nullable=False)
```

Comment:
Looking into the workaround, I am not fully sure whether to keep `bcrypt_log_rounds` in the users table, I added It because its easier to update the user password for _logged_in_ users by querying the `bcrypt_log_rounds` dynamically. Otherwise, we would have to pass the `bcrypt_log_rounds` to update method when updating password field for logged in users.
Alternatively, we can drop the `bcrypt_log_rounds` from our default configuration and use the default number (i.e 12) to hash the password, since we dont know what constitute the good hash log rounds number and 12 is good enough for adding a complexity to a hashed password.